getting-started-2.0.md

```diff
@@ -0,0 +1,152 @@
+# Getting Started with Helix
+
+This guide helps you as a web developer get started with Helix with very little effort, then progressively discover more features as you need them.
+
+## Step 1: Publish your content (look ma, no hands!)
```

Comment:
For someone who's familiar with GitHub, I think this can be simplified to
> Create a public GitHub repository with an `index.md` file in it and open to `https://<my-repo>-<my-name>.project-helix.page/` to enjoy the results. Next (or instead of creating your own repository) have a look at https://github.com/bdelacretaz/helix-example-zero which is similar to that, with headers/footers, CSS and a few other self-explaining features.

I understand the desire to explain the steps for someone who's _not_ familiar with GitHub, but for someone who is it feels like kindergarten ;-)

Not sure how best to fix this - maybe add the above short explanation at the top of this section under a "if you're familiar with GitHub and Markdown" heading?

Reply:
Or maybe we point the people who are familiar with GitHub directly to (and improved version of) https://github.com/bdelacretaz/helix-example-zero and this document is for people who are not.

Reply:
I have clarified the above ideas in the "Three main personas for documentation?" section at https://github.com/adobe/helix-home/issues/31